resources/zib/zib-VisualAcuity.xml

```diff
@@ -0,0 +1,301 @@
+          <system value="http://unitsofmeasure.org" />
+          <code value="{decimal}" />
+      </minValueQuantity>
+      <maxValueQuantity>
+        <value value="2" />
+          <system value="http://unitsofmeasure.org" />
+          <code value="{decimal}" />
+      </maxValueQuantity>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.11" />
+        <comment value="DecimalVisualAqcuity" />
+      </mapping>
+    </element>
+    <element id="Observation.bodySite">
+      <path value="Observation.bodySite" />
+      <type>
+        <code value="CodeableConcept" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation" />
+      </type>
+        <patternCodeableConcept>
+            <coding>
+                <system value="http://snomed.info/sct" />
+                <code value="81745001" />
+            </coding>
+        </patternCodeableConcept>
+    </element>
+    <element id="Observation.bodySite.extension">
+      <path value="Observation.bodySite.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.bodySite.extension:laterality">
+      <path value="Observation.bodySite.extension" />
+      <sliceName value="laterality" />
+      <max value="1" />
+    </element>
+    <element id="Observation.bodySite.extension:laterality.url">
+      <path value="Observation.bodySite.extension.url" />
+      <fixedUri value="laterality" />
+    </element>
+    <element id="Observation.bodySite.extension:laterality.value[x]">
+      <path value="Observation.bodySite.extension.value[x]" />
+      <short value="AnatomicalLocation" />
+      <definition value="The laterality of the eye of which the visual acuity has been measured." />
+      <alias value="AnatomischeLocatie" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <mapping>
+        <identity value="zib-visualacuity-v1.0-2020EN" />
+        <map value="NL-CM:12.19.5" />
+        <comment value="AnatomicalLocation" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:visualAcuityType">
+      <path value="Observation.component" />
+      <sliceName value="visualAcuityType" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:visualAcuityType.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+            <code value="16830007" />

```

Comment:
See the original issue in https://bits.nictiz.nl/browse/ZIB-1445. There are multiple concepts with the same DefinitionCode in this zib. The concept version that is added to the zib ticket actually says that this code is correct and the other one (line 80) should be changed, but maybe its best to wait for the proposal to be actually accepted and verified by the terminology team

Reply:
Since this leads to some confusion to a new reader, it's probably best to document this in the profile as well (as soon as there's clarity about the actual code to use).